src/bootstrap/v1_1/index.ts

```diff
@@ -130,26 +95,10 @@ export async function generateBootstrapTransactions({
     suggestedParams
   });
 
-  const asset2Optin =
-    assets.asset2.id === 0
-      ? null
-      : algosdk.makeAssetTransferTxnWithSuggestedParamsFromObject({
-          from: poolAddress,
-          to: poolAddress,
-          assetIndex: assets.asset2.id,
-          amount: 0,
-          suggestedParams
-        });
-
   const fundingTxn = algosdk.makePaymentTxnWithSuggestedParamsFromObject({
     from: initiatorAddr,
     to: poolAddress,
-    amount: calculatePoolBootstrapFundingTxnAmount(assets.asset2.id, {
-      liquidityTokenCreateTxn: liquidityTokenCreateTxn.fee,
-      asset1OptinTxn: asset1Optin.fee,
-      asset2OptinTxn: asset2Optin ? asset2Optin.fee : 0,
-      validatorAppCallTxn: validatorAppCallTxn.fee
-    }),
+    amount: getBootstrapFundingTxnAmountForV1(isAlgoPool),
```

Comment:
py sdk has this nice-to-have feature:
```py
required_algo = max(required_algo - pool_algo_balance, 0)
```
from Goksel:
> No it doesn’t you can calculate the pool address and send algo to it without bootstrap call.
This is nice to have, if the account is already funded. No need to fund it again.

We can implement it too:
```suggestion
    amount: Math.max(
      getBootstrapFundingTxnAmountForV1(isAlgoPool) -
        (await getAccountInformation(client, poolAddress)),
      0
    ),
```

Reply:
So this is to cover the edge case of someone using the logic sig template to create a pool address and sending ALGO to them without actually creating the pool?

Reply:
Yes AFAIU, that's what it is.